mission_control/navigator_missions/nav_missions/detect_deliver.py

```diff
@@ -11,22 +11,28 @@
 from std_srvs.srv import SetBool, SetBoolRequest
 from twisted.internet import defer
 from image_geometry import PinholeCameraModel
+from visualization_msgs.msg import Marker,MarkerArray
 import navigator_tools
 
 
 class DetectDeliverMission:
     # Note, this will be changed when the shooter switches to actionlib
-    resp = None
+    shoot_distance_meters = 5
```

Comment:
Is there a reason you don't put these in the init function? If I was looking at code and was looking for the value `self.circle_radius` which is used later in the code, I'd control-f for `self.circle_radius` which wouldn't exist.

This space is used for static class variables more so than just general parameters